.buildkite/Project.toml

```diff
@@ -55,7 +53,6 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 TerminalLoggers = "5d786b92-1e48-4d6f-9151-6b4477ca9bed"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 ThreadsX = "ac1d9e8a-700a-412c-b207-f0111f4b6c0d"
-URIs = "5c2747f8-b7ea-4ff2-ba2e-563bfd36b1d4"
 
 [compat]
 SnoopCompileCore = "2"
```

Comment:
```suggestion
SnoopCompileCore = "2, 3"
```

Reply:
Wasn't this needed?

Reply:
Not sure, I don't remember changing that

Reply:
It was needed for julia 1.11

Reply:
I forgot we haven't merged that yet